openmoc/opencg_compatible.py

```diff
@@ -1014,9 +1018,6 @@ def get_openmoc_geometry(opencg_geometry):
     opencg_geometry.assign_auto_ids()
     opencg_geometry = copy.deepcopy(opencg_geometry)
 
-    # Update Cell bounding boxes in Geometry
-    opencg_geometry.update_bounding_boxes()
```

Comment:
To be honest, I don't recall why I even had it in here, and I don't see any reason to perform this calculation before building an OpenMOC geometry. It was creating a bottleneck for a version of the BEAVRS core with over 50,000+ each of materials, cells and universes that I didn't want to wait on, so I removed it. Do you need it?